<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvertIntervals_Widget</class>
 <widget class="QWidget" name="InvertIntervals_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Invert Intervals</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_domain">
     <item>
      <widget class="QLabel" name="domain_type_label">
       <property name="text">
        <string>Define Overall Domain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="domain_type_combobox">
       <item>
        <property name="text">
         <string>Unbounded</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bounded</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_start">
     <item>
      <widget class="QLabel" name="bound_start_label">
       <property name="text">
        <string>Bound Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="bound_start_spinbox">
       <property name="minimum">
        <double>-100000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_end">
     <item>
      <widget class="QLabel" name="bound_end_label">
       <property name="text">
        <string>Bound End</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="bound_end_spinbox">
       <property name="minimum">
        <double>-100000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
